cert_manager/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG - cert_manager
 
+## 2.2.1 / 2022-09-16
+
+
```

Comment:
Not sure what to add in the changelog here since we moved this integration from -extras to -core. We also switched to OpenMetrics v2, so maybe something related to this? cc @davidor @ofek

Reply:
For the migration to core, you may take inspiration from https://github.com/DataDog/integrations-core/blob/cd4cd84e0a2f50769db2a6bf8280b88628a6fa8c/calico/CHANGELOG.md?plain=1#L9
```suggestion
* [Added] Move cert_manager to core. See [#12344](https://github.com/DataDog/integrations-core/pull/12344).
```

Reply:
Thanks @alopezz!